how-to-go-home-early-github.md

```diff
@@ -0,0 +1,73 @@
+# How to automate your workflow and go home early (Pull-requests)
+
+_If you have more than zero co-workers you are probably using some kind of platform to help you with pull requests and reviews. If your platform of choice is GitHub, you're in luck. We compiled a list of tools and tricks that can help you get your commits merged in that sweet master branch as fast as possible._
+
+### Set up a pull request template
+
+Writing a good pull request template can help you and teammates find a common ground on how to summarize what's in a PR. You can also add add checklist if there are some requirements that have to be met before it can be merged.
+
+![Alt screenshot of example template](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/pr-template.png)
+
+### Auto label your pull requests
+
+With the new GitHub actions you can automatically label your pull requests based on what files you touched in your PR. This can help a lot with the discoverability of your pull request overview.
+
+![Alt screenshot of GitHub action on timeline](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/labeler-timeline.png)
+
+To set this up, go to your GitHub repo with actions enabled, click on "New workflow" and once in the marketplace scroll down to "Automate every step in your process" where you'll find the "labeler" action:
+
+![Alt screenshot of GitHub action in marketplace](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/labeler-marketplace.png)
+
+### Automatically request reviews with "Code owners" 
+
+We can take the automatic labeling of your PR one step further. There is a feature in GitHub called "Code owners" which let's you assign co-workers to certain parts of your application(s) GitHub will use codeowners to automatically asign reviews.
+
+![Alt screenshot of reviewers being assigned](https://raw.githubusercontent.com/MyOnlineStore/blogs/how-to-go-home-early/public/codeowners.png)
+
+Setting up code owners is easy, check [this guide](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) to get started with code owners.
+
+### Let a panda spam your co-workers into submission for reviews.
+
+One thing that helped our team a lot was the addition of [Pull panda](https://pullreminders.com/). It is a slack integration that automatically sends a message to your coworkers when someone requests their review. 
```

Comment:
Use of 'co-workers' and 'coworkers'. Both are fine (a.f.a.i.k.) but I think the consistent use of either one is better.